app/controller/sdl/Abstract/Controller.js

```diff
@@ -1043,6 +1043,11 @@ SDL.SDLController = Em.Object.extend(
         SDL.SDLNonMediaModel.set('currentAppId', null);
       }
       if (app.webEngineApp && app.policyAppID in SDL.SDLModel.webApplicationFramesMap) {
+        for(var key in SDL.SDLModel.webApplicationFramesMap) {
```

Comment:
@YarikMamykin does not make any sense as the same element will be removed at line 1056

Reply:
Actually without this one in case of app unregister another app will have a black screen!